### DISCUSSION TITLE
Other Players Gear Invisible

### DISCUSSION THREAD

**Discussion by Tabzyy** [TrinityCore/TrinityCore#27912] [Q&A]
Hello! I have an issue where other players on my server appear to have no gear.  I've heard this was an old issue and wasn't able to find a solution, if anyone has any information it'd be greatly appreciated!

**Comment by Aokromes** [comment]
relog usualy "fixes" the bug, there is no permanent fix.

**Comment by CraftedRO** [comment]
https://github.com/TrinityCore/TrinityCore/issues/15159
